src/components/Cards/BountyEntryCard.tsx

```diff
@@ -147,6 +149,13 @@ export function BountyEntryCard({ data, currency, renderActions }: Props) {
                       width={IMAGE_CARD_WIDTH}
                       className={classes.image}
                       wrapperProps={{ style: { height: 'calc(100% - 60px)' } }}
+                      anim={
+                        image.type === 'video' &&
+                        videoMetadata?.duration &&
+                        videoMetadata.duration > MAX_ANIMATION_DURATION_SECONDS
+                          ? false
+                          : undefined
```

Comment:
I suggest we create a helper method for this. Will be hard to keep track across multiple files